x-pack/plugins/security_solution/public/resolver/mocks/resolver_tree.ts

```diff
@@ -309,3 +312,24 @@ export function mockTreeWithNoProcessEvents(): ResolverTree {
     },
   };
 }
+
+export function mockTreeWithNoAncestorsAndTwoChildrenAndRelatedEventsOnOrigin({
+  originID,
+  firstChildID,
+  secondChildID,
+}: {
+  originID: string;
+  firstChildID: string;
+  secondChildID: string;
+}) {
+  const baseTree = mockTreeWithNoAncestorsAnd2Children({
+    originID,
+    firstChildID,
+    secondChildID,
+  });
+  const withRelatedEvents: Array<[string, string]> = [
+    ['registry', 'access'],
```

Comment:
:grey_question: I was hoping to make this a little nore reusable, e.g. being able to pass in the set of related events from the test,

Reply:
i was hoping for the opposite :) let's discuss in office hours?

Reply:
we followed up. README with recommendations for writing mocks will be added next PR